packages/mobile/src/escrow/saga.ts

```diff
@@ -165,15 +167,19 @@ function* getSentPayments() {
     const sentPaymentsNotifications: EscrowedPayment[] = []
     for (let i = 0; i < sentPayments.length; i++) {
       const payment = sentPayments[i]
+      const recipientPhoneNumber = recipientsPhoneNumbers[sentPaymentIDs[i].toLowerCase()]
       const transformedPayment: EscrowedPayment = {
         senderAddress: payment[1],
-        recipient: recipientsPhoneNumbers[sentPaymentIDs[i]],
+        recipientPhone: recipientPhoneNumber,
         paymentID: sentPaymentIDs[i],
         currency: SHORT_CURRENCIES.DOLLAR, // Only dollars can be escrowed
         amount: payment[3],
         timestamp: payment[6],
         expirySeconds: payment[7],
       }
+      if (recipientPhoneNumberToContact[recipientPhoneNumber]) {
```

Comment:
This approach is totally fine but just a suggestion for another way to handle this would be to add this line to the object definition just above:
`recipientContact: recipientPhoneNumberToContact[recipientPhoneNumber] || undefined`